### PR TITLE
[IMPROVE] Automatically resolve conflicts on Apps-Engine versions used in manifest and package files

### PR DESCRIPTION
**Pull request by thassiov**
…son`

### CODE REVIEW

File: src/misc/folderDetails.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import Command from '@oclif/command';
 import { IAppInfo } from '@rocket.chat/apps-engine/definition/metadata';
 import chalk from 'chalk';
+import * as compareVersions from 'compare-versions';
```
Comment (graywolf336) [inline src/misc/folderDetails.ts:4]: You can do this with the `semver` package, no need to bring in yet another library. Right?

File: src/misc/folderDetails.ts [hunk 2]
```diff
@@ -97,6 +99,36 @@ export class FolderDetails {
         }
     }
 
+    public async matchAppsEngineVersion(): Promise<void> {
```
Comment (graywolf336) [inline src/misc/folderDetails.ts:102]: Shouldn't we ask them which version they prefer? Or just make the decision for them?

File: src/misc/folderDetails.ts (hunk 1 above)
Comment (thassiov) [reply src/misc/folderDetails.ts:4]: Yes, you are totally right

File: src/misc/folderDetails.ts (hunk 2 above)
Comment (thassiov) [reply src/misc/folderDetails.ts:102]: Yes, right now we are forcing this new version

Comment (d-gubert) [reply src/misc/folderDetails.ts:104]: I agree. If they realize they want a different Apps-Engine version, they can run NPM install again with that version (and deal with the potential compiler errors)

**Review by d-gubert — APPROVED** [review]

**Pull request merged**